test/acceptance/behave/components/common_steps/general_steps.py

```diff
@@ -365,8 +365,8 @@ def verify_headers_in_response(context):
     verify headers in response
```

Comment:
Note that requirements.txt referes to `requests` verion 2.7.0 (https://github.com/telefonicaid/fiware-orion/blob/develop/test/acceptance/behave/requirements.txt#L5). PR body mentions 2.11.1. Shoul requirements.txt be updated in this PR?